src/zh/zaimanhua/src/eu/kanade/tachiyomi/extension/zh/zaimanhua/Zaimanhua.kt

```
@@ -101,6 +104,15 @@ class Zaimanhua : HttpSource(), ConfigurableSource {
 
     private fun isValid(token: String): Boolean {
         if (token.isBlank()) return false
+        val parts = token.split(".")
+        if (parts.size != 3) return false
```

Should throw exception instead of returning false.